vendor/bat-native-confirmations/src/bat/confirmations/internal/platform_helper.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BAT_CONFIRMATIONS_INTERNAL_PLATFORM_HELPER_H_
+#define BAT_CONFIRMATIONS_INTERNAL_PLATFORM_HELPER_H_
+
+#include <string>
+
+#include "base/macros.h"
+#include "base/memory/singleton.h"
+#include "build/build_config.h"
+
+namespace confirmations {
+
+class PlatformHelper {
+ public:
+  static PlatformHelper* GetInstance();
```

Comment:
Wondering why do we need the whole class and hierarchy, and not just a free function with many impls (like `GetChannelName()`)?

Reply:
Fixed, thanks